### ISSUE TITLE
Large rift looks very strange in multiplayer

### ISSUE THREAD

**Issue by Sixmorphugus** [DimensionalDevelopment/DimDoors#79]
I'm using optifine so this might be a rendering bug, but here's a video:
https://dl.dropbox.com/s/jwm279qoqr0cce2/2018-07-02_21-46-15.webm?dl=0

The rift has gotten huge due to logarythmic growth (as far as I know it is intended) and is shaking around all over the place. It looks more like a flat billboard rattling around at mach 5 than a rift.

**Comment by ZombieHDGaming** [comment]
Yeah, that's definitely not what it actually does or looks like. See if removing Optifine fixes that, but we haven't gotten reports of something like this before

**Comment by Sixmorphugus** [comment]
I tried removing optifine, but it's not fixed. I'm assuming either there's a conflict with another mod or something happening with my GPU?

![Mods](https://imgur.com/ynM89iN)

**Comment by Sixmorphugus** [comment]
Yeah, here it is happening with everything disabled but dimensional doors:
https://dl.dropbox.com/s/b8v12tayndplf12/2018-07-02_22-46-02.webm?dl=0

Not sure what's going on. Is the version on curseforge old?

**Comment by Sixmorphugus** [comment]
My specs also, from dxdiag:
![computerspecs](https://i.imgur.com/IYautnC.png)
![gpuspecs](https://i.imgur.com/nJMjKdu.png)
